### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by verabehr**
* update libwasm instructions
* remove instructions for upgrading testnet chain-id

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,46 +88,15 @@ make build
...
-```
+systemctl stop palomad
 
-6. Start paloma.
+wget https://github.com/CosmWasm/wasmvm/releases/download/v1.2.3/libwasmvm.x86_64.so
+sudo mv libwasmvm.x86_64.so /usr/lib/
 
-```
-service palomad start
+systemctl start palomad
```
Comment (taariq) [inline README.md:100]: I think we want cache cleaning hygiene here as well.

**Review by taariq — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -71,7 +71,12 @@ wget -O - https://github.com/palomachain/paloma/releases/download/v1.0.0/paloma_
 sudo chmod +x /usr/local/bin/palomad
 
 # Required until we figure out cgo
-sudo wget -P /usr/lib https://github.com/CosmWasm/wasmvm/raw/main/internal/api/libwasmvm.x86_64.so
+systemctl stop palomad
+
+wget https://github.com/CosmWasm/wasmvm/releases/download/v1.2.3/libwasmvm.x86_64.so
+sudo mv libwasmvm.x86_64.so /usr/lib/
+
+systemctl start palomad
```
Comment (taariq) [inline README.md:81]: @verabehr let's also remove this line. Not needed as below we give instructions to start paloma.

File: README.md (hunk 1 above)
Comment (taariq) [inline README.md:100]: I think we can remove this one too?

File: README.md (hunk 2 above)
Comment (verabehr) [reply README.md:81]: I'll rearrange this section. As far as I understand it, this is really a prerequisite and is not directly part of installing the paloma binary

**Pull request merged**